tests/Doctrine/Tests/Mocks/ConnectionMock.php

```diff
@@ -113,6 +136,16 @@ public function setFetchOneResult($fetchOneResult)
     }
 
     /**
+     * @param \Exception $exception
```

Comment:
Please, update this docblock in order to reflect that this param also accepts `null`.